docs/templates/includes/operation_description.j2

```diff
@@ -0,0 +1 @@
+The {{ name }} operation handles configuration related to {{ model_name | get_link_to_model_page_by_name(model_name) }} model.
```

Comment:
Probably, a space is needed at the end (or in the templates where description is included):
![screen shot 2018-12-12 at 12 09 15 pm](https://user-images.githubusercontent.com/4347159/49862541-0bbd9480-fe07-11e8-9fd1-24d78e571ecb.png)

Reply:
fixed